WebFrontend/index.html

```diff
@@ -33,6 +33,47 @@
 				</menu>
 			</li>
 
+			<li class="rake state"><input type="radio" id="rake-form" name="menu">
+				<label for="rake-form" title="Select placement of spikes in rake.">
+					<img src="icons/regular-rake-solid.svg" alt="rake form">
+				</label>
+				<menu>
+					<li><input type="radio" id="rake-form-displaced" name="rake-form" checked>
+						<label for="rake-form-displaced" title="Rake in a >regular< pattern.">
+							<img src="icons/regular-rake-solid.svg" alt="displaced">
+						</label>
+					</li>
+					<li><input type="radio" id="rake-form-square" name="rake-form">
+						<label for="rake-form-square" title="Square grid rake.">
+							<img src="icons/square-rake-solid.svg" alt="grid">
+						</label>
+					</li>
+					<li><input type="radio" id="rake-form-custom" name="rake-form">
+						<label for="rake-form-custom" title="Customized rake.">
+							<img src="icons/question-solid.svg" alt="custom">
+						</label>
+					</li>
```

Comment:
Rakes are not build as grids afaik. I saw at most two rows while searching for rakes. See this [picture](https://i1.wp.com/www.ibookbinding.com/wp-content/uploads/2014/09/Marbling-Combs-and-Rakes.jpg). Maybe just remove this buttons for the demo version or rename it to spacing or something like this.